app/src/main/java/com/door43/translationstudio/ui/home/TargetTranslationAdapter.java

```diff
@@ -39,15 +42,29 @@
  */
 public class TargetTranslationAdapter extends BaseAdapter implements ManagedTask.OnFinishedListener {
     private final Context mContext;
-    private TargetTranslation[] mTranslations;
+    private List<TargetTranslation> mTranslations;
     private OnInfoClickListener mInfoClickListener = null;
     private Map<String, Integer> mTranslationProgress = new HashMap<>();
     private List<String> mTranslationProgressCalculated = new ArrayList<>();
     private List<ViewHolder> holders = new ArrayList<>();
+    private SortProjectColumnType mSortProjectColumn = SortProjectColumnType.bibleOrder;
+    private SortByColumnType mSortByColumn = SortByColumnType.projectThenLanguage;;
+
+    private static List<String> bookList = Arrays.asList(
+            "gen" , "exo", "lev", "num", "deu", "jos", "jdg", "rut",
```

Comment:
I've noticed a number of these book lists in the code. Can you abstract them all into a single location?